internal/ir/xds.go

```diff
@@ -391,6 +391,8 @@ type HTTPRoute struct {
 	ExtensionRefs []*UnstructuredRef `json:"extensionRefs,omitempty" yaml:"extensionRefs,omitempty"`
 	// Circuit Breaker Settings
 	CircuitBreaker *CircuitBreaker `json:"circuitBreaker,omitempty" yaml:"circuitBreaker,omitempty"`
+	// Connection timeout Settings
+	BackendTimeout *Timeout `json:"backendTimeout,omitempty" yaml:"backendTimeout,omitempty"`
```

Comment:
prefer if this was renamed to `Timeout` and we moved the existing `Timeout` field into it and renamed it to `RequestTimeout` under `HTTP`